git-hooks/scripts/pre-commit/lint-phpcs

```diff
@@ -1,12 +1,8 @@
 #!/bin/sh
 
-# Lint php and css code.
-
-# We only lint files from custom modules.
-# We ignore generated css files (styles*.css), sass and javascript files which
-# are linted separately, images and XSL files.
-include='^html/(modules|themes)/custom'
-exclude='(\.(js|ico|png|jpg|gif|svg|scss|xsl)$)'
+# Lint PHP files
+include='^html/(modules|themes)/custom/.*\.php$'
```

Comment:
Shouldn't this also include *.module and *.install files?

Reply:
oh you're right. I modified the `include` on accident I think. I was trying to avoid having it lint `.yml` and so forth

Regardless of expertise, would you mine following the testing steps I provided and confirm that it works? I will upload a screenshot to compare our locals.

Reply:
<img width="1680" alt="GHI-20-reference" src="https://user-images.githubusercontent.com/254753/118128216-e29bf680-b3fa-11eb-8dd1-9b89c531bec0.png">

## Incorrect OCHA Services placement

<img width="1226" alt="GHI-20-OCHA-services-incorrect" src="https://user-images.githubusercontent.com/254753/118128308-019a8880-b3fb-11eb-875f-a1cd74a8a363.png">

Reply:
Hm, you won't like that. So I did:
- checkout this branch
- composer install
- drush cim -y
- cd subtheme & nvm use & npm install & npm run sass:build

The result is different than both of your screenshots, certainly doesn't look like what you intended, and looks broken. Maybe, if I did block placements on my local instance before (which I don't really recall), would that explain it?

The result looks like this:
<img width="1790" alt="Bildschirmfoto 2021-05-13 um 17 12 03" src="https://user-images.githubusercontent.com/205433/118146669-1f71e880-b40f-11eb-9abb-be84ec6e8f67.png">

The block configuration looks like this:
<img width="934" alt="Bildschirmfoto 2021-05-13 um 17 15 54" src="https://user-images.githubusercontent.com/205433/118146718-2e589b00-b40f-11eb-963f-243a27b96e87.png">

Reply:
Only asking because neither I nor you listed this as a step: `drush cr` anywhere in there? Maybe it's just an old set of twig templates rendering the newer DB content.

The blocks are looking identical when I compare to my local admin UI. I suppose there could be differing config inside each block, but if the config sync was successful I would expect it to be the same.

Reply:
@berliner @rupl 
I think the composer.json and/or composer.lock file is missing from this PR so it's likely using whatever this https://github.com/UN-OCHA/ghi-site/blob/main/composer.json#L45 ends up being in the composer.lock file (the last 3.x.x tag) https://github.com/UN-OCHA/ghi-site/blob/main/composer.lock#L8798

If you run `composer update unocha/common_design` do you get confirmation that it is pulling the latest base theme tag `4.0.5`? Otherwise maybe remove the composer.lock file and run `composer install` again after changing the composer.json entry to `"unocha/common_design": "^4.0.5"`. 

The composer.lock should be committed.

Reply:
Yes, I cleared the cache.

Reply:
@left23 I'll try that.

Reply:
@rupl Can you commit your composer.lock?

Reply:
@rupl After you committed your `composer.lock`, running the composer install and then a `drush cr`I get this error due to the "Compatibility with Drupal 9", so I think you need to remove that too.
```
$ fin drush cr
In InfoParserDynamic.php line 53:
  Unable to parse themes/contrib/common_design/common_design.info.yml Duplicate key "base theme" detected at line 7 (near "base theme: stable").  
```

Reply:
@berliner that's easy to fix. just remove the duplicate line referenced in the message. In the mean time I filed https://humanitarian.atlassian.net/browse/CD-291 to address the error upstream, /cc @left23

Reply:
I can't reproduce this locally. I noticed there was an empty block called OCHA Services which was placed in the Header Top region. If there were menu items in that, it might be the reason it displays the way @berliner sees it.

Reply:
Thanks, I think I got overzealous since other blocks did need to be added during setup. I originally had a problem that I saw OCHA Services in the footer and I believe I tried to move it with the block system before finding the real problem, that an HTML ID was missing and causing the JS to not fire. I corrected it by modifying the CD JS and setting it as a subtheme-specific library here:

https://github.com/UN-OCHA/ghi-site/pull/12/files#diff-92b165c326b1c0ebde9d2d5f362018062255c7c15bad2d215d7b36b2af7baa71R4-R23